examples/csv/src/main.rs

```diff
@@ -298,6 +298,7 @@ mod tests {
     }
 
     /// Here, we test that the `default` variant in enums operates as expected.
+    #[cfg(not(feature = "io"))]
```

Comment:
It's not acceptable for deserialization to fail if the io feature is enabled. Can you identify the root cause for why this test fails in this case?